README.md

```diff
@@ -1 +1,140 @@
-# drupal-11-template
+# Drupal 11 TEMPLATE
+
+This is a Drupal project template for starting new projects. The template
+includes default setup and a starting point for:
+
+- PHP Code sniffer
+- PHP Code styles
+- Github actions
+- Tasks
+- Local settings
+- Docker and overrides
+- Readme
+- Changelog
+
+## Usage
+
+### Build new project from template
+
+Build a new project using this template
+
+``` shell
+gh repo create itk-dev/{PROJECT-NAME} --template itk-dev/drupal-11-template --public --clone 
+```
+
+### Prune the project
+
+Prune the project to remove/override template specific files and setup the
+project, with docker config and settings.
+
+``` shell
+task template-prune PROJECTNAME={PROJECTNAME}
+```
+
+This will:
+
+- Remove TEMPLATE_THEME and cause coding-standards-check to fail until a new
+theme is added.
+- Add/override .env and .task.env files with files that uses {PROJECTNAME}
+- Add/override a settings.local.php to web/sites/default folder
+- Add/override a services.local.yml to web/sites/default folder
+- Override CHANGELOG.md
+- Override Taskfile.yml
+
+### Finish
+
+- Replace {PROJECT-NAME} placeholder in this README.md and CHANGELOG.md
+
+<!-- markdownlint-disable MD025 -->
+
+### ---- After building a project delete this line and all above ----
+
+# Readme for {PROJECT-NAME}
+
+## Build assets
+
+### Site installation
+
+Run the following commands to set up the site a new. This will start contaners
+and run composer install, add a settings.php file run and site-install.
+
+``` shell name="site-up"
+task build-site:new
+```
+
+If the site has existing config and a settings.php file build the site from that.
+
+``` shell name="site-up"
+task build-site:existing-conf
+```
+
+When the installation is completed, that admin user is created and the password for logging in the outputted. If you
+forget the password, use drush uli command to get a one-time-login link (not the uri here only works if you are using
+trafik).
+
+``` shell name="site-login"
+itkdev-docker-compose drush user:login
+```
+
+### Access the site
+
+If you are using out `itkdev-docker-compose` simple use the command below to åbne the site in you default browser.
+
+``` shell name="site-open"
+itkdev-docker-compose open
+```
+
+Alternatively you can find the port number that is mapped nginx container that server the site at `http://0.0.0.0:PORT`
+by using this command:
+
+``` shell
+open "http://$(docker compose port nginx 8080)"
+```
```

Comment:
Consider adding a `open` (or `site:open`) task for this (and maybe a `site:url` helper task).

Until we start making real open source projects, we can assume that our https://github.com/itk-dev/devops_itkdev-docker setup is running, and use `COMPOSE_DOMAIN` from `.env` to get the site URL (cf. my comment on adding a `drush` task).

Reply:
Good idea, for future development